package.json

```
@@ -1,54 +1,42 @@
 {
+  "type": "RESTApp",
   "name": "restapp-boilerplate",
-  "version": "2.0.1",
+  "version": "1.0.0",
   "description": "A RESTApp boilerplate",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/strt/restapp-boilerplate.git"
-  },
-  "scripts": {
-    "setup": "node util/setup.js",
-    "zip": "npm run transpile && node util/zip.js",
-    "deploy": "npm run zip && node util/deploy.js",
-    "force-deploy": "npm run zip && node util/deploy.js force",
-    "create-addon": "node util/create-addon.js",
-    "sign": "npm run zip && node util/sign.js",
-    "prod-deploy": "node util/prod-deploy.js",
-    "transpile": "npx babel ./src -d ./compiled --copy-files --delete-dir-on-start",
-    "precommit": "lint-staged"
-  },
   "license": "MIT",
+  "repository": "github:strt/restapp-boilerplate.git",
+  "homepage": "https://github.com/strt/restapp-boilerplate#readme",
+  "bugs": {
+    "url": "https://github.com/strt/restapp-boilerplate/issues"
+  },
   "author": "Strateg Marknadsföring AB",
   "contributors": [
     "Alexander Karlsson <alexander.karlsson@strateg.se>",
     "Alexander Nanberg <alexander.nanberg@strateg.se>",
     "Jesper Johansson <jesper.johansson@strateg.se>"
   ],
-  "bugs": {
-    "url": "https://github.com/strt/restapp-boilerplate/issues"
+  "scripts": {
+    "build": "sitevision-scripts build",
+    "setup": "sitevision-scripts setup",
+    "deploy": "sitevision-scripts deploy",
+    "create-addon": "sitevision-scripts create-addon",
+    "sign": "sitevision-scripts sign",
+    "prod-deploy": "echo 'todo'"
```

```suggestion
    "prod-deploy": "sitevision-scripts deploy --production"
```

Implement new `--production` flag added in `0.3.0`.

---

Actually, might be better to just remove this and force the use to simply run `deploy --production`?